samples/snippets/csharp/tour/arrays/arrays.csproj

```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <DebugType>portable</DebugType>
+    <AssemblyName>arrays</AssemblyName>
+    <OutputType>Exe</OutputType>
+    <PackageId>arrays</PackageId>
+    <PackageTargetFallback>$(PackageTargetFallback);dnxcore50</PackageTargetFallback>
+  </PropertyGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netcoreapp1.0' ">
+    <PackageReference Include="Microsoft.NETCore.App" Version="1.0.3" />
```

Comment:
You probably get a warning on this line when you try to build the project. The reference to the `Microsoft.NETCore.App` metapackage is implicit with csproj.